docs/json/radarr/naming/radarr-naming.json

```diff
@@ -2,12 +2,12 @@
   "folder": {
       "default": "{Movie CleanTitle} ({Release Year})",
       "plex": "{Movie CleanTitle} ({Release Year}) {imdb-{ImdbId}}",
-      "emby": "{Movie CleanTitle} ({Release Year})",
+      "emby": "{Movie CleanTitle} ({Release Year}) [imdbid-{imdbid}]",
       "jellyfin": "{Movie CleanTitle} ({Release Year}) [imdbid-{ImdbId}]"
   },
   "file": {
       "default": "{Movie CleanTitle} {(Release Year)} {imdb-{ImdbId}} {edition-{Edition Tags}} {[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{[Mediainfo VideoCodec]}{-Release Group}",
-      "emby": "{Movie CleanTitle} {(Release Year)} - {Edition Tags }{[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{[Mediainfo VideoCodec]}{-Release Group}",
+      "emby": "{Movie CleanTitle} {(Release Year)} [imdbid-{imdbid}] - {Edition Tags }{[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{[Mediainfo VideoCodec]}{-Release Group}",
```

Comment:
Is imdbid case sensitive? If so this might need to be [imdbid-{ImdbId}]

Reply:
> Is imdbid case sensitive? If so this might need to be [imdbid-{ImdbId}]

according to the emby instructions they use `imdbid`
![image](https://github.com/TRaSH-Guides/Guides/assets/6155095/a4431014-816c-4c08-bf6e-8a50f34df832)

Reply:
I mean that for Plex naming we have `{imdb-{ImdbId}}`, and for Emby we have `[imdbid-{imdbid}]`. It's the second of the two `imdbid`s.

Reply:
![image](https://github.com/TRaSH-Guides/Guides/assets/6155095/72b1b0a1-b200-463b-9f32-999ef8a24a5c)
yeah you're right radarr's token is `{ImdbId}` but i think it doesn't care to be honest, but better to be consistent